agent.yaml

```
@@ -0,0 +1,165 @@
+version: "1.0"
+commands:
+  generate_unit_tests:
+    description: "Generate comprehensive unit tests for a given source file, creating or updating test files as needed."
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)